Fw/Time/Time.cpp

```diff
@@ -81,10 +81,20 @@ namespace Fw {
     }
 
     SerializeStatus Time::deserialize(SerializeBufferBase& buffer) {
-        // Use TimeIntervalValue's built-in deserialization
+        // Use TimeValue's built-in deserialization
         return this->m_val.deserialize(buffer);
     }
 
+    SerializeStatus Time::serializeTo(SerializeBufferBase& buffer) const {
+        // Deprecated method - calls new interface for backward compatibility
+        return this->serialize(buffer);
```

Comment:
It'll be easier to update later if we have the deprecated method call the new serializeTo, rather than the other way around here